dsa_actionkit/contexts/account_contexts.py

```diff
@@ -1,318 +1,320 @@
 import datetime
+
 from . import lib
 
+
 class subscriptions(list):
     list_ids = [1, 3, 7]
 
 
 contexts = {
-    'user_view.html': {
-        'filename': 'user_view.html',
-        'page': {
-        'title': 'User self serve',
-        'name': 'user view',
+    "user_view.html": {
```

Comment:
are you using a formatter for these is it in a pre-commit hook or something? if not can we put a little about it in the readme so we don't have reformat stuff a bunch?

(I feel like someone mentioned adding precommit hooks in one of the meetings but now I forget sorry)

Reply:
Used reference in new issue to follow up on this [here](https://github.com/dsa-ntc/actionkit-templates/issues/12): 

![image](https://github.com/dsa-ntc/actionkit-templates/assets/21191435/7a991853-8bb5-4d77-b7ba-a5e3682497ab)

Please feel free to use this feature liberally :)